grizli/grismconf.py

```diff
@@ -506,6 +506,7 @@ def get_config_filename(instrume='WFC3', filter='F140W',
         String path of the configuration file.
 
     """
+
```

Comment:
there is nothing different but this newline, can you just remove it so it doesn't show up as a diff?

Reply:
Fixed here: baf53da0ca6435830be3f5f34ef1e7dbf0e20011

Reply:
Could it be that we lost this change with the other synchronization problems?  It still shows up as a diff here.